src/types.js

```diff
@@ -23,6 +23,7 @@ export type Node = {
     next?: string,
     prev?: string,
     title: string,
+    order?: number,
```

Comment:
Not sure whether this is necessary.

Reply:
Not really, since the `order` frontmatter field is only used in the GraphQL query (and not in any Flow-covered code).